Toolset/palettes/revdatagridlibrary/behaviorsdatagridbuttonbehavior.livecodescript

```diff
@@ -68,6 +68,112 @@ local sTableObjectsA
 local sScrollerId
 local sPollID
 
+--------------------------------------------------------------------------------
+-- DataGrid 2 Variables
+
+local sEditMode
+
+---
+
+local sReorderInProgress
+local sReorderControl
+local sReorderStartIndex
+local sReorderStartLine
+local sReorderLastHoverLine
+local sReoderMouseMoveInProgress
```

Comment:
Not a bug as you've been consistent, but there's a typo in this variable name

Reply:
Haha, reoder. I'm an idiot.